### ISSUE TITLE
Add option to specify Il2Cpp module name

### ISSUE THREAD

**Issue by Fanixtar** [vfsfitvnm/frida-il2cpp-bridge#288]
Because some game have different name

**Comment by vfsfitvnm** [comment]
It's definitely something I'm going to implement; by the way what's your platform?
Meanwhile, this should do the trick:
```ts
Reflect.defineProperty(Il2Cpp, "moduleName", { value: "whatever" });

Il2Cpp.perform(() => {

});
```

**Comment by vfsfitvnm** [comment]
Implemented in `v0.8.7`
